release_notes/40-release-notes.adoc

```diff
@@ -455,6 +456,21 @@ Release date: 18 May 2023
 * Fixed issue with manifest-based secured cluster installations for {rh-rhacscs-first}.
 * Fixed issue with unwanted logging for {product-title-managed-short}.
 
+[id="resolved-in-version-402"]
+=== Resolved in version 4.0.2
+
+Release date: 31 May 2023
+
+//bugs are not linked because ACS Jira is not public
+* This release of {product-title-short} includes a fix for:
+
+** link:https://access.redhat.com/security/cve/cve-2023-24540[CVE-2023-24540], by building {product-title-short} with updated Golang.
```

Comment:
This bullet talks about how it was fixed and other 2 talk about what is fixed. May be you can keep it consistent to tell either what the fix is or what is fixed. I would choose latter.